charts/templates/coordinator.yaml

```
@@ -0,0 +1,132 @@
+          - name: EDG_COORDINATOR_SEAL_DIR
+            value: "{{ .Values.coordinator.sealDir }}"
+          - name: OE_SIMULATION
+            value: "{{ .Values.coordinator.simulation }}"
+          - name: DCAP_LIBRARY
+            value: "{{ .Values.coordinator.dcapQpl }}"
+          name: coordinator
+          image: "{{ .Values.global.image.repository }}/coordinator:{{ .Values.global.image.version }}"
+          imagePullPolicy: {{ .Values.global.image.pullPolicy }}
+          livenessProbe:
+            httpGet:
+              path: /status
+              scheme: HTTPS
+              port: {{ .Values.coordinator.clientServerPort }}
+            initialDelaySeconds: 10
+          readinessProbe:
+            failureThreshold: 7
+            httpGet:
+              path: /status
+              scheme: HTTPS
+              port: {{ .Values.coordinator.clientServerPort }}
+          ports:
+            - containerPort: {{ .Values.coordinator.clientServerPort }}
+              name: http
+            - containerPort: {{ .Values.coordinator.meshServerPort }}
+              name: grcp
+
+          resources:
+          {{- toYaml .Values.coordinator.resources | nindent 12 }}
+          volumeMounts:
+          - name: coordinator-pv-storage
+            mountPath: /coordinator/data
+          {{ if .Values.dcap }}
+          - name: dcap-conf
+            mountPath: /etc/sgx_default_qcnl.conf
+            subPath: sgx_default_qcnl.conf
+          {{ end }}
+      volumes:
+        - name: coordinator-pv-storage
+          persistentVolumeClaim:
+            claimName: coordinator-pv-claim
+        {{ if .Values.dcap }}
+        - name: dcap-conf
+          configMap:
+            name: coordinator-dcap-config
+        {{ end }}
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: coordinator-pv-claim
+  namespace: {{ .Release.Namespace }}
+spec:
+  {{- with .Values.coordinator.storageClass }}
+  storageClassName: {{ .Values.coordinator.storageClass }}
+  {{ end }}
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Mi
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: coordinator-client-api
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: LoadBalancer
+  selector:
+    {{ .Values.global.coordinatorComponentLabel }}: coordinator
+    {{ .Values.global.coordinatorNamespaceLabel }}: {{ .Release.Namespace }}
+  ports:
+  - name: http
+    port: {{ .Values.coordinator.clientServerPort }}
+    targetPort: {{ .Values.coordinator.clientServerPort }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: coordinator-mesh-api
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: ClusterIP
+  selector:
+    {{ .Values.global.coordinatorComponentLabel }}: coordinator
+    {{ .Values.global.coordinatorNamespaceLabel }}: {{ .Release.Namespace }}
+  ports:
+  - name: grcp
+    port: {{ .Values.coordinator.meshServerPort }}
+    targetPort: {{ .Values.coordinator.meshServerPort }}
```

```suggestion
    targetPort: {{ .Values.coordinator.meshServerPort }}
    appProtocol: tcp
```

Recent changes to the helm charts for OpenServiceMesh compatibility.